### PR TITLE
Add content-disposition response header configuration support with some query parameters

### PR DESCRIPTION
**Pull request by stephane-klein**
Add content-disposition response header configuration support with some query parameters.

### CODE REVIEW

**Comment by stephane-klein** [comment]
@Its-Alex need feedback + review

File: index.js [hunk 1]
```diff
@@ -25,9 +26,13 @@ const serve = ({endPoint, port, useSSL, accessKey, secretKey, bucketName}) => {
 
     return async function serve(ctx, next) {
         if (ctx.request.method === 'GET') {
-            const fileInBucket = ctx.request.url.split('?')[0];
-            let dataStream;
+            const splittedUrl = ctx.request.url.split('?');
+            const fileInBucket = splittedUrl[0];
+            const queryParams = splittedUrl.length > 1 ? parse(splittedUrl[1]) : {};
+
+            ctx.set('Content-Disposition', `${queryParams.contentDisposition ||  'attachment' }; filename="${queryParams.filename || fileInBucket}"`);
```
Comment (Its-Alex) [inline index.js:33]: @stephane-klein If this is set to `attachment` by default files will be downloaded by default

Comment (stephane-klein) [reply index.js:33]: @Its-Alex yes I know, but I need to define a default value, `inline` or `attachment`.

Do you prefer `inline` by default?

Comment (Its-Alex) [reply index.js:33]: @stephane-klein I think that by default chrome/firefox try to open a file, so I would put `inline` here to keep default behavior, what do you think?

**Comment by Its-Alex** [comment]
@stephane-klein Good that works :+1:

File: index.js (hunk 1 above)
Comment (Its-Alex) [reply index.js:33]: @stephane-klein For now we will let it like that

**Pull request merged**